.changeset/flat-cobras-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@remix-run/node": minor
+---
+
+Add "remove" and "getFilePath" methods to fileUploadHandler
```

Comment:
The exported class is called `NodeOnDiskFile`, so I think we should reference that name instead.

```suggestion
Add `getFilePath`& `remove` methods to `NodeOnDiskFile`
```